packages/block-editor/src/components/block-list/index.js

```diff
@@ -90,6 +91,7 @@ function BlockList( {
 				className
 			) }
 			{ ...__experimentalContainerProps }
+			style={ style }
```

Comment:
Do you think there's a way we could remove this div or maybe pass a `Block.*` to InnerBlocks to render it as the wrapper of the current block?

Reply:
I'm not sure I'm following... That's already the case? `Container` is variable and set by the block through `InnerBlocks`.

Reply:
And this div element is needed and cannot be removed. Or do you mean the style prop? I do agree that all props should be passed down and not just the style prop.

Reply:
I mean I'm a container block and I want to be a "section" and I don't want this extra div added by InnerBlocks so I was wondering if I could do:

```
<InnerBlocks tagName={ Block.section } />
```

Maybe it's already possible and there's something I'm missing

Reply:
Like this? https://github.com/WordPress/gutenberg/pull/19910/files#diff-3878fabdd6add9b09483511e7ff0d9b3R70

Reply:
haha I guess I missed it for some reason :)